README.md

```diff
@@ -1,31 +1,47 @@
-[![Publish To NPM](https://github.com/palmetto/palmetto-components/workflows/Publish%20To%20NPM/badge.svg)](https://github.com/palmetto/palmetto-components/actions?query=workflow%3A%22Publish+To+NPM%22)
-[![Chromatic](https://github.com/palmetto/palmetto-components/workflows/Chromatic/badge.svg?branch=master)](https://github.com/palmetto/palmetto-components/actions?query=workflow%3AChromatic)
-[![Test Coverage](https://api.codeclimate.com/v1/badges/e61e897623b87d91d155/test_coverage)](https://codeclimate.com/github/palmetto/palmetto-components/test_coverage)
+<p align="center">
+  <a href="https://palmetto.com" rel="noopener" target="_blank"><img width="150" src="https://alchemy.palmetto.com/public//default_logo/1541461188.png" alt="Palmetto logo"></a></p>
+</p>
 
-# Palmetto Components
-A React component library to power all Palmetto UI.
+<h1 align="center" style="border-bottom: none;">Palmetto Component Library</h1>
+<h3 align="center">
+
+  A [React](https://reactjs.org/) component library to power all Palmetto UI
+
+</h3>
+
+<div align="center" style="margin-top: 25px">
+
+  [![Publish To NPM](https://github.com/palmetto/palmetto-components/workflows/Publish%20To%20NPM/badge.svg)](https://github.com/palmetto/palmetto-components/actions?query=workflow%3A%22Publish+To+NPM%22) 
+  [![Chromatic](https://github.com/palmetto/palmetto-components/workflows/Chromatic/badge.svg?branch=master)](https://github.com/palmetto/palmetto-components/actions?query=workflow%3AChromatic) 
+  [![Test Coverage](https://api.codeclimate.com/v1/badges/e61e897623b87d91d155/test_coverage)](https://codeclimate.com/github/palmetto/palmetto-components/test_coverage)
+
+</div>
 
 ## Quick Start
 
 ### 1. Install
+
 `yarn add @palmetto/palmetto-components`
 
-### 2. import global scss/css
+### 2. import global css
+
 ```
-/* main.css or main.scss */
-@import '~@palmetto/palmetto-components/dist/scss/utilities.scss';
-@import '~@palmetto/palmetto-components/dist/scss/variables.scss';
+import '@palmetto/palmetto-components/dist/css/utilities.css';
```

Comment:
@JCarmine these should still carry the `@` before the import statement.

Reply:
Actually, I see now it could be both depending on where it is imported. I'm fine to leave as is for now, but just want to clarify in the future that `@import` refers to importing these in a css (or scss) file, and the regular `import` is used for importing it into something like `App.jsx`.

Reply:
Ah, hadn't noticed that detail. Logistics-app just imported straight into App.js (where I copied the lines from). Wonder if these are even functional and being used in the app then? @juanfabrega Were you part of this integration?